qt/python/mantidqt/widgets/codeeditor/execution.py

```diff
@@ -7,7 +7,7 @@
 #  This file is part of the mantidqt package
 #
 #
-from __future__ import (absolute_import, unicode_literals)
```

Comment:
Removing `unicode_literals` hides a PyCharm type warning on `mode=COMPILE_MODE` being passed `unicode` instead of `str`